src/core/workload.py

```diff
@@ -146,6 +146,15 @@ def run_bin_command(self, bin_keyword: str, bin_args: list[str], opts: list[str]
         """
         ...
 
+    @abstractmethod
+    def get_version(self) -> str:
```

Comment:
**praise** yes, we should definitely make this as part of standard workload API, IMHO. So people don't even forget to implement this and the version setting (to a bit larger extent)